### DISCUSSION TITLE
I'm really confused.....can you help me thank you.

### DISCUSSION THREAD

**Discussion by TalhaDiab** [flet-dev/flet#1911] [Q&A] [answered]
### Question

How to make NavigationRail extend from right to left?
Thank you all.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by plugeit** [comment]
you should use navigation bar for that purpose

**Reply by TalhaDiab** [reply]
can it extend from right to left?
can you provide me with  sample?

**Comment by farshadz1997** [comment, marked as answer]
take a look at this [example](https://github.com/flet-dev/examples/tree/main/python/community/responsive_menu_layout)

**Comment by TalhaDiab** [comment]
Yeah...Thank you very much .
I had seen it actually..
but the icons appear at the left side and it expand from left to right.
The best way I think is to set the the lebel in the bottom of icons.
Just it.
Thank you again.


في الاثنين، ٩ تشرين الأول، ٢٠٢٣ ١٢:٢٩ ص Farshad Zargari <
***@***.***> كتب:

**Reply by plugeit** [reply]
you should mark the answer

**Reply by TalhaDiab** [reply]
No...this doesn't answer ...
I need to make it expand from right to left and icons appear in the right
I saw I will use another method not *it's perfect*

**Reply by plugeit** [reply]
no i am asking to mark @farshadz1997 response as answer
